packages/lodestar/src/chain/chain.ts

```diff
@@ -242,6 +242,12 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
   }
 
   private processAttestation = async (latestState: BeaconState, attestation: Attestation, attestationHash: Hash) => {
+    const currentSlot = getCurrentSlot(this.config, latestState.genesisTime);
```

Comment:
latestState.slot?

this will always be slot 0

Reply:
@wemeetagain should our fork choice track it's own time? Not sure how often is onTick called in spec